core/kernel/impl/class.ApiModelOO.php

```diff
@@ -101,21 +100,8 @@ public function exportXmlRdf($sourceNamespaces = [])
      */
     public function importXmlRdf($targetNameSpace, $fileLocation)
     {
-        $returnValue = (bool) false;
-
-        if (!file_exists($fileLocation) || !is_readable($fileLocation)) {
-            throw new common_Exception("Unable to load ontology : $fileLocation");
-        }
-        
-        if (!preg_match("/#$/", $targetNameSpace)) {
-            $targetNameSpace .= '#';
-        }
-        $modFactory = new core_kernel_api_ModelFactory();
-        $returnValue = $modFactory->createModel($targetNameSpace, file_get_contents($fileLocation));
-        
-
-
-        return (bool) $returnValue;
+        $importer = $this->getServiceLocator()->get(RdfImporter::class);
```

Comment:
Are we dropping `targetNamespace` support? If yes, shouldn't be interface changed too?

Reply:
yes, normally we should change the signature , but i don't want to break anything yet,
also we will have  cleanup branch